client/lib/post-normalizer/rule-safe-image-properties.js

```diff
@@ -12,6 +12,10 @@ import { makeImageURLSafe } from './utils';
 export default function safeImagePropertiesForWidth( maxWidth ) {
 	return function safeImageProperties( post ) {
 		makeImageURLSafe( post.author, 'avatar_URL', maxWidth );
+		if ( post.feed_ID === 12098 || +post.site_ID === 1065188 ) {
```

Comment:
Numbers! So magic.

Can we move these numbers out to constants and have a comment explaining why we're doing this in this particular case?